libs/schemas/src/queries/super-admin.schemas.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from 'zod';
+
+export const TotalStats = z.object({
+  numCommentsLastMonth: z.number(),
+  numThreadsLastMonth: z.number(),
+  numPollsLastMonth: z.number(),
+  numReactionsLastMonth: z.number(),
+  numProposalVotesLastMonth: z.number(),
+  numMembersLastMonth: z.number(),
+  numGroupsLastMonth: z.number(),
+});
+
+export const GetStats = {
+  input: z.object({
+    community_id: z.string().optional(),
+  }),
+  output: z.object({
+    lastMonthNewCommunities: z.array(
+      z.object({
+        id: z.string(),
+        created_at: z.date(),
+      }),
+    ),
+    totalStats: TotalStats.extend({
+      averageAddressesPerCommunity: z.number(),
+      populatedCommunities: z.number(),
+    }),
+  }),
+};
+
+export const GetTopUsers = {
+  input: z.object({}),
+  output: z.array(z.any()),
```

Comment:
Should update this output schema

Reply:
This is kind of a flex schema, col names with spaces and stuff. We can review later since it's only for super admins.